appeals/web/src/server/lib/mappers/utils/file-upload-info-to-documents.js

```diff
@@ -0,0 +1,44 @@
+import config from '@pins/appeals.web/environment/config.js';
+
+/** @typedef {import('@pins/appeals/index.js').AddDocumentsRequest} AddDocumentsRequest */
+
+/**
+ * @param {Object} params
+ * @param {string|number} params.caseId
+ * @param {number} params.folderId
+ * @param {number} [params.redactionStatus]
+ * @param {{ files: { GUID: string, name: string, documentType: string, size: number, stage: string, mimeType: string, receivedDate: string, redactionStatus: number, blobStoreUrl: string }[] }} params.fileUploadInfo - The file upload information object.
```

Comment:
I know you didn't add this in this PR, so feel free to ignore, but it might be worth moving this typedef out and defining it separately for easier reuse, as a small improvement